src/index.html

```diff
@@ -20,8 +24,356 @@
       href="styles/index.scss"
     />
   </head>
-
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#HOME">
+        <img
+          class="header__logo"
+          src="../src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link nav__link--is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops_computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
```

Comment:
Add the modifier classes for stars everywhere 
```suggestion
          <div class="stars stars--4">
```